bids_prov/tests/test_spm_parser.py

```diff
@@ -392,322 +425,322 @@ def test_dep_regex():
-                        "[0.02, 0.02, 0.02, 0.001, 0.001, 0.001, 0.01, 0.01, 0.01, 0.001, 0.001, 0.001]",
-                    ],
-                    ["eoptions.fwhm", "[7, 7]"],
-                ],
-            },
-            {
-                "@id": "niiri:spm.spatial.preproc._7ghfrrhbkVA",
-                "label": "spatial.preproc._7",
-                "used": ["niiri:GunZippedFiles4"],
-                "wasAssociatedWith": "RRID:SCR_007037",
-                "attributes": [
-                    ["channel.biasreg", "0.001"],
-                    ["channel.biasfwhm", "60"],
-                    ["channel.write", "[0, 1]"],
-                    ["tissue(1).tpm", "{'/home/radon/spm12/tpm/TPM.nii,1'}"],
-                    ["tissue(1).ngaus", "1"],
-                    ["tissue(1).native", "[1, 0]"],
-                    ["tissue(1).warped", "[0, 0]"],
-                    ["tissue(2).tpm", "{'/home/radon/spm12/tpm/TPM.nii,2'}"],
-                    ["tissue(2).ngaus", "1"],
-                    ["tissue(2).native", "[1, 0]"],
-                    ["tissue(2).warped", "[0, 0]"],
-                    ["tissue(3).tpm", "{'/home/radon/spm12/tpm/TPM.nii,3'}"],
-                    ["tissue(3).ngaus", "2"],
-                    ["tissue(3).native", "[1, 0]"],
-                    ["tissue(3).warped", "[0, 0]"],
-                    ["tissue(4).tpm", "{'/home/radon/spm12/tpm/TPM.nii,4'}"],
-                    ["tissue(4).ngaus", "3"],
-                    ["tissue(4).native", "[1, 0]"],
-                    ["tissue(4).warped", "[0, 0]"],
-                    ["tissue(5).tpm", "{'/home/radon/spm12/tpm/TPM.nii,5'}"],
-                    ["tissue(5).ngaus", "4"],
-                    ["tissue(5).native", "[1, 0]"],
-                    ["tissue(5).warped", "[0, 0]"],
-                    ["tissue(6).tpm", "{'/home/radon/spm12/tpm/TPM.nii,6'}"],
-                    ["tissue(6).ngaus", "2"],
-                    ["tissue(6).native", "[0, 0]"],
-                    ["tissue(6).warped", "[0, 0]"],
-                    ["warp.mrf", "1"],
-                    ["warp.cleanup", "1"],
-                    ["warp.reg", "[0, 0.001, 0.5, 0.05, 0.2]"],
-                    ["warp.affreg", "'mni'"],
-                    ["warp.fwhm", "0"],
-                    ["warp.samp", "3"],
-                    ["warp.write", "[0, 1]"],
-                ],
-            },
-            {
-                "@id": "niiri:spm.spatial.normalise.write._8EzpCNFeKjF",
-                "label": "spatial.normalise.write._8",
-                "used": ["niiri:ForwardDeformations7", "niiri:RealignedImages(Sess1)5"],
-                "wasAssociatedWith": "RRID:SCR_007037",
-                "attributes": [
-                    ["woptions.bb", "[-78, -112, -70, 78, 76, 85]"],
-                    ["woptions.vox", "[2, 2, 2]"],
-                    ["woptions.interp", "4"],
-                    ["woptions.prefix", "'w'"],
+RECORDS = defaultdict(list,     {
```

Comment:
Can you add a comment to explain what the `RECORDS` variable stores and what it is used for?

Reply:
I have removed `RECORDS`. This variable was used to test for equality with a parser output. Since we are now testing for inclusiveness, we don't need it anymore.